src/main/java/com/google/sps/servlets/ElectionServlet.java

```diff
@@ -98,13 +119,55 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
                 conn.disconnect();
             }
         }
+        
+        String elections = strBuf.toString(); 
+
+        JSONObject obj = new JSONObject(elections);
+        JSONArray electionData = obj.getJSONArray("elections");
+        int numElections = electionData.length();
+
+        for (int i = 0; i < numElections; ++i) {
+            JSONObject currentElection = electionData.getJSONObject(i);
+            Entity electionEntity = new Entity("Election");
+
+            /* The "id" of an Election Entity is stored as a property instead of
+               replacing the Datastore-generated ID because  Datastore may 
```

Comment:
please remove extra space.

Reply:
Are you referring to the carriage return?